lib/generic/modules/arthritis.json

```diff
@@ -0,0 +1,737 @@
+        "======================================================================",
+        " RHEUMATOID ARTHRITIS                                                 ",
+        "======================================================================",
+
+        "Source: http://www.cdc.gov/arthritis/basics/rheumatoid.htm",
+
+        "Shares the same medications and care pathways as gout."
+      ],
+      "target_encounter": "Gout_Diagnosis",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "69896004",
+          "display": "Rheumatoid arthritis"
+        }
+      ],
+      "direct_transition": "Gout_Diagnosis"
+    },
+
+    "Lupus": {
+      "type": "ConditionOnset",
+      "remarks": [
+        "======================================================================",
+        " LUPUS                                                                ",
+        "======================================================================",
+
+        "Source: http://www.cdc.gov/arthritis/basics/lupus.htm"
+      ],
+      "target_encounter": "Lupus_Diagnosis",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "200936003",
+          "display": "Lupus erythematosus"
+        }
+      ],
+      "direct_transition": "Lupus_Diagnosis"
+    },
+
+    "Lupus_Diagnosis": {
+      "type": "Encounter",
+      "encounter_class": "ambulatory",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "185347001",
+          "display": "Encounter for problem"
+        }
+      ],
+      "direct_transition": "Lupus_CarePlan"
+    },
+
+    "Lupus_CarePlan": {
+      "type": "CarePlanStart",
+      "target_encounter": "Lupus_Diagnosis",
+      "assign_to_attribute": "arthritis_careplan",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "872781000000100",
+          "display": "Musculoskeletal care"
+        }
+      ],
+      "activities": [
+        {
+          "system": "SNOMED-CT",
+          "code": "440381005",
+          "display": "Behavior to prevent sun exposure"
+        }
+      ],
+      "direct_transition": "Lupus_Nonopioid_Pain_Medication"
+    },
+
+    "Lupus_Nonopioid_Pain_Medication": {
+      "type": "MedicationOrder",
+      "target_encounter": "Lupus_Diagnosis",
+      "remarks": [
+        "A stronger NSAID pain reliever than the basic Aleve."
+      ],
+      "codes": [
+        {
+          "system" : "RxNorm",
+          "code" : "849437",
+          "display" : "Naproxen sodium 550 MG [Anaprox]"
+        }
+      ],
+      "direct_transition": "Immune_Suppressant"
+    },
+
+    "Immune_Suppressant": {
+      "type": "MedicationOrder",
+      "target_encounter": "Lupus_Diagnosis",
+      "codes": [
+        {
+          "system": "RxNorm",
+          "code": "835900",
+          "display": "cycloSPORINE, modified 100 MG [Neoral]"
+        }
+      ],
+      "direct_transition": "Terminal"
+    },
+
+    "Terminal": {
```

Comment:
As far as I can tell arthritis never really ends. It just maintains or gets worse as you age. If a patient's arthritis is bad enough to merit medication I don't think the condition or the medication to treat it should ever end.